scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1060,6 +1060,10 @@ void BPF_STRUCT_OPS(layered_enqueue, struct task_struct *p, u64 enq_flags)
 	if (vtime_before(vtime, layer->vtime_now - layer_slice_ns))
 		vtime = layer->vtime_now - layer_slice_ns;
 
+	if (!tctx->all_cpus_allowed){
+		scx_bpf_dispatch(p, SCX_DSQ_GLOBAL, slice_ns, enq_flags);
+		goto preempt;
```

Comment:
So, the problem with this is that on muti-socket machines, tasks which are affine to a NUMA node would all slip out of their layers and end up in `DSQ_GLOBAL`.

Reply:
Thanks, will see if stalls go away using SCX_DSQ_LOCAL (I think that avoids that?).

Reply:
It'd be avoid `GLOBAL` but that likewise would send all such tasks to the local CPU (and e.g. it wouldn't be able to preempt other tasks if the local CPU is currently busy).

Reply:
> wouldn't be able to preempt other tasks if the local CPU is currently busy

I'm not 1000% sold on that being the problem I want to solve (yet). 

I previously thought it was, but my test captured an issue similar looking to the one I want to solve which makes me think "maybe this is the problem instead of that", idk.